site2/docs/admin-api-topics.md

```diff
@@ -197,6 +197,8 @@ You can check the following statistics of a given non-partitioned topic.
 
   -   **storageSize**: The sum of the ledgers' storage size for this topic. The space used to store the messages for the topic.
 
+  -   **earliestMsgPublishTimeInBacklogs**: The publish time of the earliest message in the backlog.
```

Comment:
It's in format of millisecond timestamp?

Reply:
Yes, I think it's in milliseconds.

Reply:
> Yes, I think it's in milliseconds.

Great, It's better if we can add this in the doc.